api/swagger/swagger.yml

```diff
@@ -1,16 +1,134 @@
 consumes:
-- application/json
+  - application/json
 info:
   description: Thyra HTTP server API.
   title: thyra-server
   version: 0.0.0
 produces:
-- application/json
+  - application/json
 schemes:
-- http
-- https
-swagger: "2.0"
+  - http
+  - https
+swagger: '2.0'
 paths:
+  /mgmt/wallet:
+    get:
```

Comment:
I guess a getter for a specific id will be need at some point to extract the wallet.
New _good first issue_ ?